docs/standards/generic-standards/lsp1-universal-receiver.md

```diff
@@ -35,26 +45,50 @@ This standard defines a single function named `universalReceiver(...)` that coul
 
-![schema of universal receiver transaction](/img/standards/lsp1/ur-transaction.jpeg)
 
 ## Extension
 
 :::info
 
-See the **[LSP1-UniversalReceiverDelegate](../universal-profile/lsp1-universal-receiver-delegate.md)** standard for more details.
+See the **[LSP1-UniversalReceiverDelegate](../generic-standards/lsp1-universal-receiver-delegate.md)** standard for more details.
 
 :::
 
-LSP1-UniversalReceiverDelegate is an **optional extension** to the LSP1-UniversalReceiver Standard. As well as notifying a contract about the incoming and outgoing transactions via an event, it will delegate the `universalReceiver(...)` functionality to an external contract that can **handle and react to specific calls** with its custom logic.
+Customizing the `universalReceiver(..)` function is an option that users have to allow the function to **behave differently on different actions**. However, it's not advised to hardcode the logic of reacting on specific actions inside the function, as **this logic may need to change in the future** depending on several factors (eg. the vault where the tokens are forwarded get hacked, new staking contract is deployed, decided to revert on specific tokens later). 
+
+**[LSP1-UniversalReceiverDelegate](../generic-standards/lsp1-universal-receiver-delegate.md)** is an **optional extension** to the **[LSP1-UniversalReceiver](#)** standard. As well as notifying a contract about the incoming and outgoing transactions by emitting an event, it can delegate the call to an external contract that can **handle and react to specific calls** with its custom logic.
+
+![Universal Receiver Delegate contract](/img/standards/lsp1/universal-receiver-delegate.jpeg)
+
+
+The address of the **external contract** can be a variable that changes in the contract storage by a setter function. In this way, users can customize such contracts to implement a specific logic that is changeable anytime in the future.
+
+![Multiple Universal Receiver Delegate](/img/standards/lsp1/multiple-urd.jpeg)
 
-The address of the **external contract** MUST be set as a value for the **LSP1UniversalReceiverDelegate data key** shown below to enable the optional extension. This key-value pair inside the **[ERC725Y Data key-value store](https://github.com/ERC725Alliance/erc725/blob/main/docs/ERC-725.md#erc725y)** of the contract implementing the `universalReceiver(...)` function will act as a reference, making this external contract upgradeable if required.
+If the contract implementing the `universalReceiver(..)` supports **[ERC725Y Data key-value store](https://github.com/ERC725Alliance/erc725/blob/main/docs/ERC-725.md#erc725y)**, the address of the **external contract** MUST be set as a value for the **LSP1UniversalReceiverDelegate data key** shown below to enable the optional extension. This key-value pair will act as a reference, making this external contract upgradeable if required.
```

Comment:
Same changes suggested as above.

BUt if its's in two places and duplicated, maybe we can remove it in one of them?

Reply:
I would keep it, as people could read the 2 standards separately, it's not that big deal imo